neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -56,13 +56,13 @@ impl Stream for WebTransportRecvStream {
 
 impl RecvStream for WebTransportRecvStream {
     fn receive(&mut self, _conn: &mut Connection) -> Res<(ReceiveOutput, bool)> {
-        self.events.data_readable(self.get_info());
+        self.events.data_readable(&self.get_info());
```

Comment:
This `get_info()` method is creating a new instance of `Http3StreamInfo`.  That seems wasteful.  Any reason why that value can't be owned?  ... so that `get_info()` can return a reference to it.

Reply:
You mean adding a `stream_info` field, initializing that in `new`, and then returning a ref?